lib/models/application.coffee

```diff
@@ -191,15 +186,14 @@ getApplicationModel = (deps, opts) ->
 						$eq: [
 							$tolower: $: 'app_name'
 							appName
-						]
-					expand:
+						],
 						user:
```

Comment:
This can be backported right?

Reply:
I don't think it can? Before we were expanding user, so you'd get it in your result (but only the id), now we're not expanding it at all.